README.md

```diff
@@ -4,46 +4,52 @@ Deploys the Odyssey (OCM Search v2) components.
 
 ## Installing the Search Operator in a Red Hat OpenShift cluster
 
-### Prerequisites
+### 0. Prerequisites
 
 - You will need [Operator SDK](https://sdk.operatorframework.io/) to install search operator bundle image. [Download and install](<https://sdk.operatorframework.io/docs/installation/>) client version >= v1.15
 - You'll need **Red Hat Advanced Cluster Management** v2.5 or later.
-- Update the MulticlusterHub CR to disable search v1.
-    In the MulticlusterHub CR set `enabled: false` where `spec.overrides.components.name = search`
-
->    `oc edit mch -n open-cluster-management`
-
-```yaml
-    spec:
-    overrides:
-        components:
-        - enabled: false
-            name: search
-```
-
-After disabling search (v1), install the search operator (v2) in the open-cluster-management namespace.
 
-#### 1. Log into the cluster using CLI
+### 1. Log into the cluster using CLI
 
 If not done already, use `oc login` to log in to the cluster by replacing `yoururl` and `yourpassword` below
 
 ```bash
 oc login https://yoururl.com:6443 -u kubeadmin -p yourpassword
 ```
 
-#### 2. Create Image pull secret to pull images from quay (Hub cluster)
+### 2. Create Image pull secret to pull images from quay (Hub cluster)
 
 1. go to https://quay.io/user/<your_id>?tab=settings replacing <your_id>  with your username
 1. click on Generate Encrypted Password
 1. enter your quay.io password
 1. select Kubernetes Secret from left-hand menu
-1. Download yaml file and rename secret as  `search-pull-secret`
+1. Download yaml file and rename secret as `search-pull-secret`
+
+### 3a. Automatic install
+A script is provided which covers step 4 and 5 is provided in this repo for RHACM 2.6 version. You can launch it with and it will offer you the possibility to choose a version or use the default one. You will need to have the secret created in step 2. in the `deploy` folder.
```

Comment:
`You can launch it with and it will offer you the possibility` , Little rewording `You can launch the script and it will offer you the possibility`